archipelago/pnr_.py

```diff
@@ -116,7 +127,7 @@ def pnr(arch, input_netlist=None, packed_file="", cwd="", app_name="",
         if wave_filename is not None:
             shutil.copy2(wave_filename, copy_to_dir)
 
-    return placement_result, routing_result
+    return placement_result, routing_result, id_to_name
```

Comment:
Does this mean that the upstream needs to be changed?

Reply:
This is causing the error here: https://buildkite.com/stanford-aha/aha-flow/builds/5981#1c4bd7bb-c181-40bf-b72f-c488c98d9c61/501-556

Reply:
I've already changed the upstream so this should be fixed